guides/specification.md

```diff
@@ -348,6 +351,22 @@ facts:
 
 Finally, gathered facts, are used in Check's [Expectations](#expectations) to determine whether expected conditions are met for the best practice to be adhered.
 
+## Customizable
+
+Wanda's built-in checks, which include expected values, are **customizable** by **default**. Users can modify these check values through the [Trento Web console](https://github.com/trento-project/web) to adapt to specific system and environmental configurations.
+
+To explicitly mark a check as non-customizable, set the `customizable`  key to `false`:
+
+```yaml
+customizable: false
+```
+
+To explicitly mark a check as customizable, set the `customizable` key to `true`:
+
+```yaml
+customizable: false
```

Comment:
```suggestion
```

Besides it being `true` in this case 😄 , I believe it might turn out a bit redundant. What about omitting it?